pom.xml

```diff
@@ -101,15 +101,7 @@
 
         <dependency>
             <groupId>com.vaadin</groupId>
-            <artifactId>vaadin-testbench</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-        <!-- https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager -->
-        <dependency>
-            <groupId>io.github.bonigarcia</groupId>
-            <artifactId>webdrivermanager</artifactId>
-            <version>5.1.0</version>
+            <artifactId>vaadin-testbench-junit5</artifactId>
```

Comment:
Why do we want to update this starter to JUnit 5? I propose to do the JUnit upgrade in a separate branch. In Vaadin 24.0 we just provide a TB implementation for 5, but don't recommend to replace 4 with 5.

Reply:
@ZheSun88

Reply:
aww.. i cannot remember the reason.. but most likely related to the change to skeleton-starter-flow-spring.. https://github.com/vaadin/skeleton-starter-flow-spring/commit/8f4a0a73003c125e747460ee5a440bde52bb8fbf